### DISCUSSION TITLE
Is downgrading package version / switching branch possible?

### DISCUSSION THREAD

**Discussion by Sceptero** [LykosAI/StabilityMatrix#86] [Q&A] [answered]
Is it possible to downgrade existing package to lower version or switch used git branch/tag?

**Comment by ionite34** [comment]
Not currently but will be planned 👍

**Reply by hyppyhyppo** [reply]
Waiting for this feature ... What the best option to make a backup before updating ?
Simply save the content of the package folder somewhere ?
And what is the zip file in it for ?

**Reply by ScionoicS** [reply]
Would still love this feature to be available . It looks like the package manager is just git on the back end. Shouldn't be hard.

**Reply by Gh0st-drive** [reply]
Also waiting for this, with the current version stopping ComfyUI package stops all other running packages. Want to roll back

**Comment by mohnjiles** [comment, marked as answer]
<img width="413" height="303" alt="image" src="https://github.com/user-attachments/assets/a365f770-f1dd-4167-a3b5-1777d5152e24" />
This is available now by selecting the 3-dots menu -> Change Version
